.github/workflows/patches.yml

```
@@ -70,12 +70,19 @@ jobs:
           repository: FFmpeg/FFmpeg
           ref: release/4.2
           path: ffmpeg-release-4.2
+      - name: Clone libaom
+        run: git clone https://aomedia.googlesource.com/aom
+      - name: Configure libaom
+        run: |
+          [ -d ~/.ccache ] && sudo chown -R "$USER": ~/.ccache
+          cmake -S aom -B aom/build-static -DCMAKE_BUILD_TYPE=Debug -DBUILD_SHARED_LIBS=OFF -DENABLE_TESTS=0
```

```suggestion
          cmake -S aom -B aom/build-static -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -DENABLE_TESTS=0
```
Could you make this release to match the svt-av1 make above?